plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2688,3 +2688,30 @@ function wc_update_890_update_connect_to_woocommerce_note() {
 	$new_note = WooSubscriptionsNotes::get_note();
 	$new_note->save();
 }
+
+/**
+ * Disables the PayPal Standard gateway for stores that aren't using it.
+ *
+ * PayPal Standard has been deprecated since WooCommerce 5.5, but there are some stores that have it showing up in their
+ * list of available Payment methods even if it's not setup. In WooComerce 8.9 we will disable PayPal Standard for those stores
+ * to reduce the amount of new connections to the legacy gateway.
+ *
+ * Shows an admin notice to inform the store owner that PayPal Standard has been disabled and suggests installing PayPal Payments.
+ */
+function wc_update_890_update_paypal_standard_load_eligibility() {
+	$paypal = class_exists( 'WC_Gateway_Paypal' ) ? new WC_Gateway_Paypal() : null;
+
+	if ( ! $paypal ) {
+		return;
+	}
+
+	// If PayPal is enabled or set to load, but the store hasn't setup PayPal Standard live API keys and doesn't have any PayPal Orders, disable it and show a notice to Admin.
+	if ( ( 'yes' === $paypal->enabled || 'yes' === $paypal->get_option( '_should_load' ) ) && ! $paypal->get_option( 'api_username' ) && ! $paypal->has_paypal_orders() ) {
+		$paypal->update_option( '_should_load', wc_bool_to_string( false ) );
+
+		WC_Admin_Notices::add_custom_notice(
```

Comment:
This probably is not required, I am not sure if all merchants can differentiate between PayPal Standard vs extension, and will probably cause more confusion than help. Given that it's not setup anyway, and there are no orders, we can drop this silently, but accompany the release with a blog post explaining this.

Reply:
That said, if we do want to push this notice, we should check if PayPal  Payments is already installed on the site.

Reply:
I agree that this noticed isn't needed. Removed in 0934556067d0f3db0438c2d55e5d951d8983b880.